io.kopia.KopiaUI.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "io.kopia.KopiaUI",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "kopia-ui",
+    "separate-locales": false,
+    "rename-desktop-file": "kopia-ui.desktop",
+    "rename-icon": "kopia-ui",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
```

Comment:
This won't work.

Reply:
What is _this_? I am using the flatpak myself for several days without noticing any problems...

Reply:
It's used for tray icons. You should use `--own-name=org.kde.*`

Reply:
Tray icons are working for me. `org.kde.*` looks too permissive for my taste. See https://github.com/flathub/org.keepassxc.KeePassXC/pull/61/commits/aef2c6ea0a9f455eb17c1f19038e017c535f00f7 for a detailed explanation.